### PR TITLE
Fixes #8644.

### LINKED ISSUE #8644: Fixture load 2.0.4 postgis pgsql error
Since 2.0.4 (prior to that, was working fine) I'm having this error when i try to fixture/load.

My database is PostgreSQL and extends PostGIS.

What is happening is that the Global fixture InitDb is trying to execute the following SQL:
```SQL
    ... ALTER TABLE "public"."geography_columns" DISABLE TRIGGER ALL; ...
```

and I get the error:

    Exception 'yii\db\Exception' with message 'SQLSTATE[42809]: Wrong object type: 7 ERROR:  "geography_columns" is not a table

Now... the thing is that "geography_columns" isn't a table, its a **view**, and yet its included in the SQL statement.

views
(all of the views are in the sql)

Here's the stack trace:

    Stack trace:
    #0 /vagrant/src/yii2-app/vendor/yiisoft/yii2/db/Command.php(779): yii\db\Schema->convertException(Object(PDOException), 'ALTER TABLE "pu...')
    #1 /vagrant/src/yii2-app/vendor/yiisoft/yii2/test/InitDbFixture.php(94): yii\db\Command->execute()
    #2 /vagrant/src/yii2-app/vendor/yiisoft/yii2/test/InitDbFixture.php(76): yii\test\InitDbFixture->checkIntegrity(false)
    #3 /vagrant/src/yii2-app/vendor/yiisoft/yii2/test/FixtureTrait.php(114): yii\test\InitDbFixture->beforeUnload()
    #4 /vagrant/src/yii2-app/vendor/yiisoft/yii2/console/controllers/FixtureController.php(147): yii\console\controllers\FixtureController->unloadFixtures(Array)
    #5 [internal function]: yii\console\controllers\FixtureController->actionLoad('*')
    #6 /vagrant/src/yii2-app/vendor/yiisoft/yii2/base/InlineAction.php(55): call_user_func_array(Array, Array)
    #7 /vagrant/src/yii2-app/vendor/yiisoft/yii2/base/Controller.php(151): yii\base\InlineAction->runWithParams(Array)
    #8 /vagrant/src/yii2-app/vendor/yiisoft/yii2/console/Controller.php(91): yii\base\Controller->runAction('load', Array)
    #9 /vagrant/src/yii2-app/vendor/yiisoft/yii2/base/Module.php(455): yii\console\Controller->runAction('load', Array)
    #10 /vagrant/src/yii2-app/vendor/yiisoft/yii2/console/Application.php(161): yii\base\Module->runAction('fixture/load', Array)
    #11 /vagrant/src/yii2-app/vendor/yiisoft/yii2/console/Application.php(137): yii\console\Application->runAction('fixture/load', Array)
    #12 /vagrant/src/yii2-app/vendor/yiisoft/yii2/base/Application.php(375): yii\console\Application->handleRequest(Object(yii\console\Request))
    #13 /vagrant/src/yii2-app/yii(23): yii\base\Application->run()
    #14 {main}

### PR DESCRIPTION
**Pull request by ricpelo**
| Q | A |
| --- | --- |
| Is bugfix? | yes |
| New feature? | no |
| Breaks BC? | no |
| Tests pass? | yes |
| Fixed issues | #8644 |

### CODE REVIEW

File: framework/db/pgsql/Schema.php [hunk 1]
```diff
@@ -213,6 +217,50 @@ protected function findTableNames($schema = '')
     }
 
     /**
+     * Returns all views names in the database.
+     * @param string $schema the schema of the views. Defaults to empty string, meaning the current or default schema.
+     * @return array all views names in the database. The names have NO schema name prefix.
```
Comment (samdark) [inline framework/db/pgsql/Schema.php]: Need `@since 2.0.9`

Comment (ricpelo) [inline framework/db/pgsql/Schema.php]: Do I need to include it myself?

Comment (samdark) [inline framework/db/pgsql/Schema.php]: If you have time to do it, yes. If not - we can do it.

Comment (samdark) [inline framework/db/pgsql/Schema.php]: If you'll do it, please add a line for changelog as well.

Comment (ricpelo) [inline framework/db/pgsql/Schema.php]: Oh, great, actually I don't know how to change a commit from a pull request :/ . May you please do it yourself? Sorry and thanks!

Comment (samdark) [inline framework/db/pgsql/Schema.php]: Just push to the same branch, it will be updated automatically.

Comment (ricpelo) [inline framework/db/pgsql/Schema.php]: Hope I did it well.

**Comment by ricpelo** [comment]
It seems to be a conflict in CHANGELOG.md, but I'm unable to resolve it myself.

**Comment by samdark** [comment]
No worries.

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/commit/33dfddd0c2b019470243d100b9db59f58ad8367b Merged! Thank you!

**Comment by ricpelo** [comment]
Great! Thanks to you!

**Pull request closed without merging**